YouTag/YYTTagCell.swift

```diff
@@ -44,13 +40,21 @@ class YYTTagCell: UICollectionViewCell {
 	
 	fileprivate var titleLabelDefaultLeadingAnchor: NSLayoutConstraint?
 	fileprivate var titleLabelWithImageLeadingAnchor: NSLayoutConstraint?
+    fileprivate var tagList: [String]?
+    var areSuggestionsEnabled: Bool? {
+        didSet {
+            if ((tagList != nil) && (areSuggestionsEnabled!)) {
+                textField.filterStrings(tagList ?? [""])
+            } else {
+                textField.filterStrings([])
+            }
```

Comment:
Make sure indentation is proper

Reply:
Why do we have a tag list for each cell. That not needed. Also, `areSuggestionsEnabled` didSet should not filter the string but it should affect whether the suggestion tableview would appear or not. Filtering should be at textChange func for the textfield.

Reply:
This is not doing any filtering, it is simply setting the list of all possible drop downs in the SearchTextField class - the method name is a bit misleading.

Reply:
Thats really a misleading name but alright for `areSuggestionsEnabled` all good. Still the tagList should not be a member of the cell